modules/agent-install-mirroring-for-disconnected-registry.adoc

```diff
@@ -0,0 +1,253 @@
+              next-hop-address: 192.168.111.2
+              next-hop-interface: eno1
+              table-id: 254
+  EOF
+----
++
+<1> This IP address is used to determine which node performs the bootstrapping process as well as running the `assisted-service` component.
+You must provide the IP address when you do not specify the node's IP addresses in the `networkConfig` parameter. If this address is not provided, one IP address is selected from the provided nodes's `networkConfig`.
+<2> The number of hosts defined must match the total number of hosts defined in the `install-config.yaml` file, which is the sum of the values of the `compute.replicas` and `controlPlane.replicas` parameters. When 3 master nodes and 0 worker nodes are defined in the `install-config.yaml` file,
+the number of hosts defined is 3. When 3 master nodes and 2 worker nodes are defined in the `install-config.yaml` file, the number of hosts defined is 5.
+
+. Mirror the images to a directory on the removable media:
++
+[source,terminal]
+----
+$ oc adm release mirror -a ${LOCAL_SECRET_JSON} --to-dir=${REMOVABLE_MEDIA_PATH}/mirror quay.io/${PRODUCT_REPO}/${RELEASE_NAME}:${OCP_RELEASE}-${ARCHITECTURE}
+----
+
+. Upload the images to the local container registry from the removable media:
++
+[source,terminal]
+----
+$ oc image mirror -a ${LOCAL_SECRET_JSON} --from-dir=${REMOVABLE_MEDIA_PATH}/mirror "file://openshift/release:${OCP_RELEASE}*" ${LOCAL_REGISTRY}/${LOCAL_REPOSITORY} <1>
+----
++
+<1> For `REMOVABLE_MEDIA_PATH`, you must use the same path that you specified when you mirrored the images.
+
+. Create the Agent image by running the following command:
++
+[source,terminal]
+----
+$ openshift-install agent create image
+----
+
+. Boot the `agent.iso` image on the bare metal machines. You can run the image on any Linux distribution.
+
+. To track the progress and verify sucessful installation, run the following command:
++
+[source,terminal]
+----
+$ openshift-install agent wait-for install-complete
+----
++
+.Example output
+[source,terminal]
+----
+...................................................................
+...................................................................
+INFO Cluster is installed
+INFO Install complete!
+INFO To access the cluster as the system:admin user when using 'oc', run
+INFO     export KUBECONFIG=/home/core/installer/auth/kubeconfig
+INFO Access the OpenShift web-console here: https://console-openshift-console.apps.sno-cluster.test.example.com
+----
+
++
+[IMPORTANT]
+====
+If you are using the optional method of the ZTP manifests, there are two mirror configuration files: `registries.conf` and `ca-bundle.crt` file. The `registries.conf` file must be added to `mirror/ directory` for the subcommand `openshift-install create-image command` to execute sucessfully.
+
+.Sample `registries.conf` file
+[source,yaml]
+----
+[[registry]]
+location = "registry.ci.openshift.org/ocp/release"
+mirror-by-digest-only = false
```

Comment:
Although we have used used mirror-by-digest-only = false, it should be
mirror-by-digest-only = true